src/framework/test_group.ts

```diff
@@ -31,6 +31,7 @@ class Test<F extends Fixture> {
     this.cases = cases;
   }
 
+  // What does this syntax mean??
```

Comment:
`*` makes it an iterator. `object[Symbol.iterator]` is a field you can set on an object to make it be iterable with `for..of`.

Reply:
Ew, ok so this is setting the `iterator` member symbol to a function? Ack.